spec/game_spec.rb

```diff
@@ -39,15 +39,42 @@
   end
 
   it 'keeps track of player turns' do
-    game = game_setup([1, 2])
+    game = game_setup(FakeUi.new([0, 1]))
 
     2.times { game.play_turn }
 
     expect(board.player_at(1)).not_to eq(board.player_at(0))
   end
 
+  it 'gets Ui to display the board' do
+    ui = FakeUi.new([0])
+    game = game_setup(ui)
+
+    game.play_turn
+
+    expect(ui.display_count).to eq(1)
+  end
+
+  it 'get Ui to alert on invalid move' do
+    ui = FakeUi.new(['a', 1])
+    game = game_setup(ui)
+
+    game.play_turn
+
+    expect(ui.alert_count).to eq(1)
+  end
+
+  it 'gets Ui to greet' do
+    ui = FakeUi.new([0, 3, 1, 4, 2])
+    game = game_setup(ui)
+
+    game.play
+
+    expect(ui.greet_count).to eq(1)
+  end
+
   it 'plays till win' do
```

Comment:
Prefer full words, until would be better. Till is not what you think it is :) til would be the shortened version of until, but use the full word even in the description of your tests.